src/main/resources/db/changelog/db.changelog-1.0.xml

```diff
@@ -0,0 +1,279 @@
+                <constraints nullable="false"/>
+            </column>
+            <column name="deleted_at" type="TIMESTAMP WITH TIME ZONE"/>
+
+            <column name="authorisation_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="begin_date" type="DATE">
+                <constraints nullable="false"/>
+            </column>
+            <column name="end_date" type="DATE"/>
+        </createTable>
+        <addForeignKeyConstraint baseTableName="authorisation_term" baseColumnNames="authorisation_id"
+                                 referencedTableName="authorisation" referencedColumnNames="authorisation_id"
+                                 constraintName="fk_authorisation_term_authorisation"
+                                 onDelete="NO ACTION" onUpdate="NO ACTION"/>
+        <sql><![CDATA[
+            ALTER TABLE authorisation_term
+                ADD CONSTRAINT ck_authorisation_term_end_date
+                    CHECK (end_date IS NULL OR begin_date < end_date)
+            ]]></sql>
+    </changeSet>
+
+    <changeSet id="2021-11-02-create-email-table" author="terova">
+        <createTable tableName="email">
+            <column autoIncrement="true" name="email_id" type="BIGSERIAL">
+                <constraints primaryKey="true" primaryKeyName="email_pkey"/>
+            </column>
+            <column name="version" type="INT" defaultValueNumeric="0">
+                <constraints nullable="false"/>
+            </column>
+            <column name="created_by" type="TEXT"/>
+            <column name="modified_by" type="TEXT"/>
+            <column name="deleted_by" type="TEXT"/>
+            <column name="created_at" type="TIMESTAMP WITH TIME ZONE" defaultValueComputed="CURRENT_TIMESTAMP">
+                <constraints nullable="false"/>
+            </column>
+            <column name="modified_at" type="TIMESTAMP WITH TIME ZONE" defaultValueComputed="CURRENT_TIMESTAMP">
+                <constraints nullable="false"/>
+            </column>
+            <column name="deleted_at" type="TIMESTAMP WITH TIME ZONE"/>
+
+            <column name="sender" type="text">
+                <constraints nullable="false"/>
+            </column>
+            <column name="recipient" type="text">
+                <constraints nullable="false"/>
+            </column>
+            <column name="subject" type="text">
+                <constraints nullable="false"/>
+            </column>
+            <column name="body" type="text">
+                <constraints nullable="false"/>
+            </column>
+            <column name="sent" type="TIMESTAMP WITH TIME ZONE"/>
+            <column name="error" type="text"/>
+        </createTable>
+    </changeSet>
+
+    <changeSet id="2021-11-02-create-authorisation_term_reminder-table" author="terova">
+        <createTable tableName="authorisation_term_reminder">
```

Comment:
Olisiko tämä parempi olla `authorisation_term_email` taulu?

Reply:
Voisi, jos meillä on useampiakin sähköposteja voimassaoloon liittyen tämä liitostaulu voisi kertoa mistä oli kyse jolloin mainitsemasi taulun nimi olisi varmaan hyvä, mutta nyt on vain yksi, annan olla näin.